.github/workflows/release-next.yml

```
@@ -0,0 +1,133 @@
+name: Create Release of next branch
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - next
+
+jobs:
+  generate-version:
+    name: Generate Release Version
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.generate_version.outputs.version }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          ref: "next"
+
+      - name: Generate Next Release Version
+        id: generate_version
+        uses: actions/github-script@v6
```

```suggestion
        uses: actions/github-script@v7
```

is there any reason why not use the latest? 👀

---

Nope! No reason. Updated on fa976f4697fe65605001b3fdd4e5093ce29452e0